src/views/tokenLaunch/farmData.tsx

```diff
@@ -206,20 +206,20 @@ function FarmData({
               <Typography>{getFarmLiquidity(farm.index)}</Typography>
             </TableCell>
             <TableCell align="center">
-              <Typography>{getUserPoolBalanceFormated(farm.pid, farm.index)}</Typography>
+              <Typography>{ connected ? getUserPoolBalanceFormated(farm.pid, farm.index) : '-'}</Typography>
               <Typography style={{marginTop: '4px'}} color="textSecondary" variant="body2">{getUserPoolBalanceInUSD(farm.pid, farm.index)}</Typography>
```

Comment:
We only need to show this if 'connected'. Not even '-' in case not connected. It is a my stake value in terms of dollar.